antsibull_changelog/fragment.py

```diff
@@ -203,7 +204,8 @@ def load_fragments(paths: PathsConfig, config: ChangelogConfig,
         if os.path.isdir(fragments_dir):
             fragment_paths = [
                 os.path.join(fragments_dir, path)
-                for path in os.listdir(fragments_dir) if not path.startswith('.')]
+                for path in os.listdir(fragments_dir)
+                if not re.search(config.fragment_filename_ignore_re, path)]
```

Comment:
I would do this in another loop, to be able to warn about files that did not match. Also, it would be better to compile the regex before using it in a loop.